src/OrleansProviders/Streams/Memory/MemoryBatchContainer.cs

```diff
@@ -9,10 +7,10 @@
 
 namespace Orleans.Providers
 {
-    [Serializable]
     internal class MemoryBatchContainer<TSerializer> : IBatchContainer
```

Comment:
Unclear how this will work?  Batch containers are the way we send sets of events to consumers.  If these are not serializable, how will they be sent?
If they are serialized, how is the serializer assigned?